src/nupic/regions/PyRegion.py

```diff
@@ -177,6 +177,15 @@ def getOutputElementCount(self, name):
     name: the name of the output
     """
 
+  def getAlgorithm(self):
```

Comment:
This is for debugging purposes anyway, and when debugging people oftentimes try accessing `._sfdr` and `_tfdr`, so the need is there. How else can we add this functionality cleanly?